cluster/local/local.sh

```diff
@@ -24,7 +24,7 @@ function copy_image_to_cluster() {
 case "${1:-}" in
   up)
     kubectl apply -f ${scriptdir}/helm-rbac.yaml
-    helm init --service-account tiller
+    ${HELM_TOOL} init --service-account tiller
```

Comment:
@HashedDan help me understand how this `${HELM_TOOL}` var gets set.  Looking at the changes made to the upbound/build submodule, I see `HELM` defined, but not `HELM_TOOL`: https://github.com/upbound/build/compare/e66bc9d7d58cc0a4a14292a62f20d2163f9dcf3b...f99b0548684bc747ba3f71364281ff77a26c0ca0#diff-1383e81af4d5978d81cbd072297a82a4R118

Where/how is `HELM_TOOL` defined?  Thank you!

Reply:
@jbw976 thanks for pointing that out! I believe that was a remnant of my changes before updating the build submodule. The variable has been changed to `HELM` from `HELM_TOOL` in the following commit and is working properly.